### DISCUSSION TITLE
Development of SVG features

### DISCUSSION THREAD

**Discussion by chmunozpardo** [3b1b/manim#1319] [Show and tell]
I've been integrating features from SVG images like colors, arcs commands, strokes and some other compatibility issues that I've been facing.

So far this is what it can do:
https://user-images.githubusercontent.com/5047248/105612817-1f38d900-5d9d-11eb-8651-26a4052ba7ef.mp4

The most notorious problems are:
- Integration of style classes, gradients and other referenced properties from SVG commands.
- Integration of properties like fill-rule, shadows and similar.

I'm open to suggestions on this.

**Comment by sahilmakhijani** [comment]
Check this out
https://pypi.org/project/svgelements/
